src/ui/theme/componentTheme/MenuButton.ts

```diff
@@ -0,0 +1,34 @@
+'use client';
+
+import { menuAnatomy } from '@chakra-ui/anatomy';
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react';
+
+const { definePartsStyle, defineMultiStyleConfig } = createMultiStyleConfigHelpers(
+  menuAnatomy.keys
+);
+
+const baseStyle = definePartsStyle(props => ({
+  // this will style the MenuButton component
+  button: {
+    _hover: {
+      bg: 'border',
+    },
+    _active: {
+      bg: 'border',
+    },
+  },
+  // this will style the MenuItem and MenuItemOption components
+  item: {
+    padding: '12px 16px',
```

Comment:
We aim to avoid using hard coded numbers for spacing. Instead, we use ChakraUI spacing variables https://chakra-ui.com/docs/styled-system/theme